csharp/ql/src/Security Features/CWE-134/UncontrolledFormatString.ql

```diff
@@ -12,17 +12,12 @@
  */
 
 import csharp
-import semmle.code.csharp.security.dataflow.flowsources.Remote
-import semmle.code.csharp.security.dataflow.flowsources.Local
+import semmle.code.csharp.security.dataflow.flowsources.FlowSources
 import semmle.code.csharp.frameworks.Format
 import FormatString::PathGraph
 
 module FormatStringConfig implements DataFlow::ConfigSig {
-  predicate isSource(DataFlow::Node source) {
-    source instanceof RemoteFlowSource
-    or
-    source instanceof LocalFlowSource
-  }
+  predicate isSource(DataFlow::Node source) { source instanceof ThreatModelFlowSource }
```

Comment:
```suggestion
    predicate isSource(DataFlow::Node source) {
    source instanceof RemoteFlowSource
    or
    source instanceof LocalFlowSource
  }
```

Reply:
In [f488f23](https://github.com/github/codeql/pull/15419/commits/f488f23a48db82b3a4c9d9f26f430328180d7e8c) I changed it to

```suggestion
  predicate isSource(DataFlow::Node source) {
    source instanceof ThreatModelFlowSource
    or
    source instanceof LocalFlowSource
  }
```

since we're moving things over to `ThreatModelFlowSource`. If that's not what you intended, please let me know and I'll change it over.

Reply:
Oh sorry - yes this is what I meant :-)